packages/@atjson/document/src/document.ts

```diff
@@ -288,43 +317,14 @@ export class Document {
       console.error("Failed to insert text", e);
     }
 
-    /*
```

Comment:
Unrelated but removed some commented out code, not sure why this is here / hasn't been deleted

Reply:
Yeah - I think we left it in mostly as reference for future implementers (if that ever happens). If only there was some way to do non-inline documentation / annotations... 🤪 I'd personally leave it in, but not bothered either way.

Reply:
I'll put it back and make an issue to do something with commented-out code